src/v/redpanda/admin/migrations.cc

```diff
@@ -82,18 +77,11 @@ ss::httpd::migration_json::inbound_migration_state to_admin_type(
     for (auto& cg : idm.groups) {
         migration.consumer_groups.push(cg);
     }
-    migration.auto_advance = idm.auto_advance;
```

Comment:
why is auto_advance removed here, but it isn't below for output_migration handling?

Reply:
whoops, it's a typo, thank you!